src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesToObject = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((acc, curr) => {
+      const [key, value] = curr.split(':').map((str) => str.trim());
+
+      acc[key] = value;
```

Comment:
Assigning `value` to `acc[key]` without checking if `key` is defined can lead to unexpected behavior if the input string is malformed. Consider adding a validation step to ensure `key` is not undefined or an empty string.